app/javascript/components/subject/monitoring_actions/CaseStatus.js

```diff
@@ -79,7 +79,13 @@ class CaseStatus extends React.Component {
           modal_text: `The case status for the selected record will be updated to ${value} and moved to the appropriate line list in the Exposure Workflow.`,
         });
 
-        // changing case status to Unknown, Suspect or Not a Case in the exposure workflow
+        // changing case status to Unknown, Suspect or Not a Case while on the PUI line list in the exposure workflow
+      } else if (!confirmedOrProbable && !this.state.isolation && this.props.patient.public_health_action != 'None') {
```

Comment:
can you add this case to `CaseStatus.test.js` so that file maintains full coverage?

Reply:
Just pushed a commit with the new test, ready for re-review